### ISSUE TITLE
PATCH doesn't support partial updates of the data of an object (bug and/or feature?)

### ISSUE THREAD

**Issue by alextreme** [maykinmedia/objects-api#466]
### Product versie / Product version

2.4.3

### Omschrijf het probleem / Describe the bug

From @swrichards 

Hello all, a quick sanity check regarding my expectations of how PATCH should work in the Objects API. My understanding is that I should be able to supply a partial record when using PATCH, that this partial should subsequently be merged with the existing object, and that new merged object should be validated according to the object type's JSON schema. Put differently: the object in the body of a PATCH should not itself have to be valid according to the JSON schema.

TLDR: [the test I've written here should pass](https://github.com/maykinmedia/objects-api/compare/swr/partial-update-with-subset-of-valid-fields?expand=1#diff-c900604ca49840241e6d00d829c50ad83d98cbe15ce867417ffeac500322aba9R351-R360) in my view, but instead I get a 400 with:
```
(Pdb) response.json()
{'non_field_errors': ["'diameter' is a required property"]}
```

Is my understanding of the expected behavior correct?

**Comment by joeribekker** [comment]
@swrichards how blocking is this?
